### DISCUSSION TITLE
How should titled boxes be tagged?

### DISCUSSION THREAD

**Discussion by mbertucci47** [latex3/tagging-project#662] [General]
What is the correct tag structure for boxed content with a title/header? Here's an example with [tcolorbox](https://www.ctan.org/pkg/tcolorbox):
```tex
\DocumentMetadata
  {
    lang=en-US,
    pdfversion=2.0,
    pdfstandard=ua-2,
    testphase={phase-III,title,math,table,firstaid}
  }
\documentclass{article}
\usepackage{tcolorbox}
\begin{document}
Some normal text
\begin{tcolorbox}[title=Some title]
Some boxed text
\end{tcolorbox}
\end{document}
```
<img width="243" alt="tcolorbox_struc" src="https://github.com/user-attachments/assets/c3cc7058-232d-46e1-b05a-ecb49203f56b">

The title content and the box content are tagged identically.

Since a title like this does not fall under the `<H1>`, etc. headings, what is the correct tagging here? Should the title be tagged as `<Caption>` like a theorem heading?
```tex
\DocumentMetadata
  {
    lang=en-US,
    pdfversion=2.0,
    pdfstandard=ua-2,
    testphase={phase-III,title,math,table,firstaid}
  }
\documentclass{article}
\newtheorem{theorem}{Theorem}
\begin{document}
Some normal text
\begin{theorem}[Some heading]
Some theorem text
\end{theorem}
\end{document}
```
<img width="243" alt="theorem_struc" src="https://github.com/user-attachments/assets/1db65411-6317-48ae-b1c1-2b3ebe25d6ae">

(Also, is `<Sect>` or `<Div>` better here? The [pdfa best practices](https://pdfa.org/wp-content/uploads/2019/06/TaggedPDFBestPracticeGuideSyntax.pdf) say `<Div>` is a division "without semantic intent". If a box has a title, I'd argue it has semantic intent. Not sure about untitled boxes.)

This is also relevant for e.g. [framed](https://www.ctan.org/pkg/framed)'s `titled-frame` environment.

**Comment by car222222** [comment]
I sssume that here you mean a visual box containing text?

**Comment by car222222** [comment]
But is it not best to assume that anything like  boxing some text has semantic intent?  
When would use of such a box have no semantic intent (and thus be meaningless)?

**Comment by FrankMittelbach** [comment]
by default, yes I guess. but there are certainly cases where boxes are there just for layout reasons in which case yu want to have the "box" structure come out as nonstruct. However, as a default I guess Sect and Caption would be the right tags for a box with a "title" as offered by tcolorbox
